CollapseLauncher/Classes/GameManagement/GameSettings/Genshin/RegistryClass/GeneralData.cs

```diff
@@ -174,104 +200,124 @@ internal class GeneralData
         /// </summary>
         public int audioDynamicRange { get; set; } = 0;
 
-        public bool firstHDRSetting { get; set; } = true;
-        public int mtrTraceCount { get; set; } = 5;
-        public bool disableRewiredInitProtection { get; set; } = false;
-        public string greyTestDeviceUniqueId { get; set; } = "";
-        public bool mtrUseOldWinVersion { get; set; } = false;
-        public List<int> _serializedCodeSwitches { get; set; }
-        public int urlCheckAbortTimeOutCount { get; set; } = 3;
-        public int urlCheckSueecssTraceCount { get; set; } = 5; //yes, that is actually the class name, its not a typo by us...
-
         /// <summary>
-        /// This define "<c>Music Volume</c>" slider in-game.<br/>
-        /// Valid values: 0-10
-        /// Default: 10
+        /// This defines "<c>Audio Output</c>" combo box in-game.
+        /// Valid Values: 0 (stereo), 1 (surround)
+        /// Default: 0
         /// </summary>
-        public int volumeMusic { get; set; } = 10;
+        public int audioOutput { get; set; } = 0;
 
         /// <summary>
-        /// This holds value for controllers input that has been customized.
+        /// Audio related stuff...
```

Comment:
"Audio parameters used by Unity for Android devices" or something like that. Specificity is always a good thing in cases like these.

Reply:
same as above

Reply:
Made it clearer as of https://github.com/neon-nyan/Collapse/commit/f2f38dd39f11b69fdc06ac1e2bca0530f6571b40